homeassistant/components/rituals_perfume_genie/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "rituals_perfume_genie",
+  "name": "Rituals Perfume Genie",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/rituals_perfume_genie",
+  "requirements": [
+    "pyrituals==0.0.2"
+  ],
+  "codeowners": [
```

Comment:
Is this device discoverable via `dhcp` ?

https://developers.home-assistant.io/docs/creating_integration_manifest/#dhcp

Reply:
No. It uses a Espressif chip with default MAC Address and hostname `ESP_XXXXXX`. It could be any devices using that chip.